recipes/guessmylt/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "guessmylt" %}
+{% set version = "0.2.5" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 8139301bae16c424c06f61a8da1db50798646b1202379930621a4eb129d0bbcf
+
+build:
+  number: 0
+  noarch: python
+  entry_points:
+    - GUESSmyLT = GUESSmyLT.GUESSmyLT:main
+    - GUESSmyLT-example = GUESSmyLT.test.example:main
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv"
+
+requirements:
+  host:
+    - python>3
+    - bcbio-gff ==0.6.4
+    - biopython ==1.67
+    - pip
+    - pysam >=0.13.0
+    - snakemake ==5.4.*
+    - trinity ==2.8.4
+    - busco ==3.0.2
+    - bowtie2 >=2
+  run:
+    - python>3
+    - git >=2.11.1
+    - bcbio-gff ==0.6.4
+    - biopython ==1.67
```

Comment:
As @HadrienG said - not too happy with the specific pin.

Here's the problem: If you install both guessmylt and packageB and packageB requires bioython ==1.66 then conda will (after churning for an hour) tell you it can't install.

That's why we prefer ranges. If you require a package that has a history of breaking changes, you can use things like `>=1.67,<1.68.0a0` to keep it to whatever upstream appears to use for bug fixes only.

Reply:
I did some test and it sounds to work fine. The problem I had in the past was with python 2.7 anyway. So, I  have relaxed the required versions.